Modules/_hashopenssl.c

```diff
@@ -1700,18 +1696,19 @@ _hashlib_HMAC_copy_impl(HMACobject *self)
 }
 
 static void
-_hmac_dealloc(HMACobject *self)
+_hmac_dealloc(PyObject *op)
 {
+    HMACobject *self = HMACobject_CAST(op);
     PyTypeObject *tp = Py_TYPE(self);
     HMAC_CTX_free(self->ctx);
     PyObject_Free(self);
     Py_DECREF(tp);
 }
 
 static PyObject *
-_hmac_repr(HMACobject *self)
+_hmac_repr(PyObject *self)
 {
-    PyObject *digest_name = py_digest_name(HMAC_CTX_get_md(self->ctx));
+    PyObject *digest_name = _hashlib_hmac_get_name(self, NULL);
```

Comment:
Wouldn't this prepend an extra `"hmac-"`?

Reply:
Another reason why we avoid changing something across the whole codebase. You never know what kind of test coverage there is. Or what kind of assumptions are hiding in an overlooked comment.